testing/daml/Testing/Init/Init.daml

```diff
@@ -51,11 +51,13 @@ data PvpContracts =
     payment : ContractId SettlementInstruction
     delivery : ContractId SettlementInstruction
 
-allocateAndInitDemo : Script InitData
+
+allocateAndInitDemo : Script PartiesForTesting
 allocateAndInitDemo = script do
   -- Participants set up parties, connecting to the two Banking domains (EUR/GBP)
   parties@PartiesForTesting{..} <- setupParties
   initDemo parties
+  return parties
```

Comment:
One can use `return` but there was a convention from Spider to rather use `pure` (e.g. that's less similar to Java's return which is absolutely different).

Reply:
FYI we tried to (mostly) follow this rule.

Reply:
addressed